azure-pipelines.yml

```diff
@@ -0,0 +1,199 @@
+trigger:
+  tags:
+    include:
+      - v*
+  branches:
+    include:
+      - master
+
+pr:
+  branches:
+    include:
+      - master
+
+variables:
+  - group: website_secrets
+  - name: KAFKA_VERSION
+    value: 1.1
+  - name: COMPOSE_FILE
+    value: docker-compose.1_1.yml
+
+####### Linter
+jobs:
```

Comment:
It's not something we need to do within this PR, but I wonder if we would see any benefit from not starting Kafka and Zookeeper in each job, but rather define them as container resources: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema#container-resource

It would be really neat if they could be re-used across each job, but I don't know if that's how that works, or if it'll boot all the resources in each job anyway.

Reply:
Yeah, I considered that, but I already had a lot of changes. We can explore in another PR.